Marlin/src/gcode/control/M80_M81.cpp

```diff
@@ -103,7 +103,10 @@ void GcodeSuite::M81() {
 
   #if HAS_SUICIDE
     stepper.synchronize();
-    suicide();
+    #ifndef suicide
+	#define suicide
+	#endif
```

Comment:
No tabs. Two spaces for indentation.

Reply:
Ok, filed for future reference. Will fix after work.